src/steps/render.js

```diff
@@ -55,29 +55,51 @@ export default async function render(state, req, res) {
   ]);
 
   // add meta
+  // (this is so complicated to keep the order backward compatible to make the diff tests happy)
```

Comment:
Is this comment really necessary? 😉

Reply:
yes. otherwise my co-coders complain about why not just use `...meta.custom`